packages/engineer/src/cli-commands.ts

```diff
@@ -115,6 +115,8 @@ export const startCommand: Command = (program) =>
                     nodeEnvironmentsMode,
                 });
 
+                process.on('SIGINT', dispose);
```

Comment:
watchClose is only called on graceful close of webpack compiler, it made sense for me to handle SIGINT in the cli to gracefully cleanup what I can @RomanYarik

Reply:
I like. works on all 3 platforms?

Reply:
https://nodejs.org/api/process.html#process_signal_events

```'SIGINT' from the terminal is supported on all platforms, and can usually be generated with Ctrl+C (though this may be configurable). It is not generated when terminal raw mode is enabled and Ctrl+C is used.```